src/postfixCompletionProvider.ts

```diff
@@ -86,7 +86,28 @@ export class PostfixCompletionProvider implements vsc.CompletionItemProvider {
 
     const binaryExpression = this.findClosestParent(node, ts.SyntaxKind.BinaryExpression) as ts.BinaryExpression;
 
-    if (binaryExpression && binaryExpression.operatorToken.kind !== ts.SyntaxKind.EqualsToken) {
+    if (binaryExpression && ![
+      ts.SyntaxKind.EqualsToken,
+      ts.SyntaxKind.PlusEqualsToken,
+      ts.SyntaxKind.MinusEqualsToken,
+      ts.SyntaxKind.SlashEqualsToken,
+      ts.SyntaxKind.AsteriskEqualsToken,
+      ts.SyntaxKind.AsteriskAsteriskEqualsToken,
+      ts.SyntaxKind.AmpersandEqualsToken,
+      // Bitwise assignments
+      ts.SyntaxKind.BarEqualsToken,
+      ts.SyntaxKind.BarBarEqualsToken,
+      ts.SyntaxKind.CaretEqualsToken,
+      ts.SyntaxKind.LessThanLessThanToken,
+      ts.SyntaxKind.LessThanLessThanEqualsToken,
+      ts.SyntaxKind.GreaterThanEqualsToken,
+      ts.SyntaxKind.GreaterThanGreaterThanEqualsToken,
+      ts.SyntaxKind.GreaterThanGreaterThanGreaterThanEqualsToken,
+      // relativly new
+      ts.SyntaxKind.AmpersandAmpersandEqualsToken,
+      ts.SyntaxKind.QuestionQuestionToken,
+      ts.SyntaxKind.BarBarEqualsToken,
+    ].includes(binaryExpression.operatorToken.kind)) {

```

Comment:
Omg, so many of them :D
Can you extract that to a private `isAssignmentOperator` method?

Reply:
All right, I'll handle that as I have small free time slot now and I'd like to release a hotfix with that today.